### ISSUE TITLE
scoped style in Vue component not reflected in extension's browser instance 

### ISSUE THREAD

**Issue by corey-cosman** [microsoft/vscode-chrome-debug#1006]
- VS Code Version: 1.44.2
- Log file (set `"trace": true` in launch config): I set trace: true in launch.json (where to find log file?)

Steps to reproduce:

This may or may not be Vue related: I don't have a ton of time to do a proper investigation across multiple frameworks, I want to see if anyone can reproduce:

1. start basic vue package with vue cli (or use existing)
2. `npm i` and `npm start`
3. create a dummy component that simply renders a div with the text `test`, then add a dummy class `eg: <div class="test">test</div>` to this 
4. add a scoped style to this component and set background to some color: 
eg:
```
 <style scoped>
.test {
background: tomato;
 </style>
```
5. add this to your launch.json (fixes a bug with vue and this extension, regarding breakpoints jumping) :

```
{
  "version": "0.2.0",
  "configurations": [
    {
      "type": "chrome",
      "request": "launch",
      "name": "Launch Chrome against localhost",
      "url": "http://localhost:8080/",
      "webRoot": "${workspaceFolder}/src",
      "trace": true,
      "sourceMapPathOverrides": {
        "webpack:///./*": "${webRoot}/*",
        "webpack:///src/*": "${webRoot}/*",
        "webpack:///*": "*",
        "webpack:///./~/*": "${webRoot}/node_modules/*",
        "webpack:///./src/*": "${webRoot}/*"
      },
      "sourceMaps": true
    }
  ]
}

```

6. launch vscode debugger (Launch Chrome against localhost)

7. confirm or deny that this dummy component style of `background: tomato` is not being rendered in this browser

8. confirm or deny that when you remove the scoped style, style is being rendered

9. confirm or deny that when you add scoped back to the style tag, and then instead, go to the div and add: `<div  style=" background: tomato;">test</div>`  that this renders the style properly

10. confirm or deny that when you have the scoped style and class setup as per the test from #7, that this works properly outside of this extension (by just accessing localhost in your broswer)

**Comment by roblourens** [comment]
I don't think your issue has anything to do with this extension. It just launches Chrome and your page should be loaded in basically the same way as usual. I don't know much about Vue so I'm not sure what the issue would be.

**Comment by corey-cosman** [comment]
Yea it may just be an issue with this extension's compatibility with the Vue framework and possibly others (or maybe its something else), but I opened this issue to try and determine if it could be reproduced by other users of this extension in the vue context (and that of other frameworks as well). I apologize if this issue does not belong here, I have realized that there are simple workflows around this it, so its not blocking in anyway, and therefore not pressing. Just figured I would make an attempt at reaching out for guidance with understanding the root of it. Thanks
